ThunderSVM GCC 6 or later

Ubuntu 17.10
cuda 9.1 (installed with .deb)

Error occurs with or without -j in make:

```
-- The C compiler identification is GNU 7.2.0
-- The CXX compiler identification is GNU 7.2.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Warning (dev) at /usr/share/cmake-3.9/Modules/FindOpenMP.cmake:200 (if):
  Policy CMP0054 is not set: Only interpret if() arguments as variables or
  keywords when unquoted.  Run "cmake --help-policy CMP0054" for policy
  details.  Use the cmake_policy command to set the policy and suppress this
  warning.

  Quoted variables like "c" will no longer be dereferenced when the policy is
  set to NEW.  Since the policy is not set the OLD behavior will be used.
Call Stack (most recent call first):
  /usr/share/cmake-3.9/Modules/FindOpenMP.cmake:324 (_OPENMP_GET_FLAGS)
  CMakeLists.txt:26 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.

Compile with CUDA
-- Found Threads: TRUE
-- Configuring done
-- Generating done
-- Build files have been written to: /home/devon/Code/thundersvm/build
[  3%] Building NVCC (Device) object src/thundersvm/CMakeFiles/thundersvm.dir/kernel/thundersvm_generated_smo_kernel.cu.o
In file included from /usr/local/cuda/include/host_config.h:50:0,
                 from /usr/local/cuda/include/cuda_runtime.h:78,
                 from <command-line>:0:
/usr/local/cuda/include/crt/host_config.h:121:2: error: #error -- unsupported GNU version! gcc versions later than 6 are not supported!
 #error -- unsupported GNU version! gcc versions later than 6 are not supported!
  ^~~~~
CMake Error at thundersvm_generated_smo_kernel.cu.o.Release.cmake:222 (message):
  Error generating
  /home/devon/Code/thundersvm/build/src/thundersvm/CMakeFiles/thundersvm.dir/kernel/./thundersvm_generated_smo_kernel.cu.o


src/thundersvm/CMakeFiles/thundersvm.dir/build.make:70: recipe for target 'src/thundersvm/CMakeFiles/thundersvm.dir/kernel/thundersvm_generated_smo_kernel.cu.o' failed
make[2]: *** [src/thundersvm/CMakeFiles/thundersvm.dir/kernel/thundersvm_generated_smo_kernel.cu.o] Error 1
CMakeFiles/Makefile2:126: recipe for target 'src/thundersvm/CMakeFiles/thundersvm.dir/all' failed
make[1]: *** [src/thundersvm/CMakeFiles/thundersvm.dir/all] Error 2
Makefile:83: recipe for target 'all' failed
make: *** [all] Error 2
```

--- a/README.md
+++ b/README.md
@@ -13,20 +13,20 @@
 
 ## What's new
 - We have recently released [ThunderGBM](https://github.com/Xtra-Computing/thundergbm), a fast GBDT and Random Forest library on GPUs.
-- add scikit-learn interface, see [here](https://github.com/zeyiwen/thundersvm/tree/master/python)
+- add scikit-learn interface, see [here](https://github.com/Xtra-Computing/thundersvm/tree/master/python)
 - pre-built binaries and DLL for Windows x64 on CPUs are [avaliable](https://ci.appveyor.com/project/shijiashuai/thundersvm/branch/master/artifacts)
 ## Overview
 The mission of ThunderSVM is to help users easily and efficiently apply SVMs to solve problems. ThunderSVM exploits GPUs and multi-core CPUs to achieve high efficiency. Key features of ThunderSVM are as follows.
 * Support all functionalities of LibSVM such as one-class SVMs, SVC, SVR and probabilistic SVMs.
 * Use same command line options as LibSVM.
-* Support Python, R and Matlab interfaces.
+* Support [Python](python/), [R](R/) and [Matlab](Matlab/) interfaces.
 * Supported Operating Systems: Linux, Windows and MacOS.
 
 **Why accelerate SVMs**: A [survey](https://www.kaggle.com/amberthomas/kaggle-2017-survey-results) conducted by Kaggle in 2017 shows that 26% of the data mining and machine learning practitioners are users of SVMs.
 
 [Documentations](docs/index.md) | [Installation](docs/how-to.md#install-thundersvm) | [API Reference (doxygen)](http://Xtra-Computing.github.io/thundersvm/)
 ## Contents
-- [Getting Started](https://github.com/zeyiwen/thundersvm#getting-started)
+- [Getting Started](https://github.com/Xtra-Computing/thundersvm#getting-started)
 - [Working without GPUs](docs/get-started.md#working-without-gpus)
 ## Getting Started
 
@@ -38,6 +38,7 @@ The mission of ThunderSVM is to help users easily and efficiently apply SVMs to 
 If you want to use GPUs, you also need to install CUDA.
 
 * [CUDA](https://developer.nvidia.com/cuda-downloads) 7.5 or above
+
 ### Quick Install
 Download the Python wheel file (For Python3 or above).
 
@@ -65,7 +66,7 @@ clf.fit(x, y)
 ```
 ### Download
 ```bash
-git clone https://github.com/zeyiwen/thundersvm.git
+git clone https://github.com/Xtra-Computing/thundersvm.git
 ```
 ### Build on Linux (build [instructions](docs/get-started.md#installation-for-macos) for MacOS and Windows)
 ##### ThunderSVM on GPUs
@@ -73,6 +74,13 @@ git clone https://github.com/zeyiwen/thundersvm.git
 cd thundersvm
 mkdir build && cd build && cmake .. && make -j
 ```
+
+If you run into issues that can be traced back to your version of gcc, use `cmake` with a version flag to force gcc 6. That would look like this:
+
+```bash
+cmake -DCMAKE_C_COMPILER=gcc-6 -DCMAKE_CXX_COMPILER=g++-6 ..
+```
+
 
 ##### ThunderSVM on CPUs
 ```bash

